### ISSUE TITLE
CA expiration

### ISSUE THREAD

**Issue by Foresaken1** [RasPlex/rasplex-installer#52]
So as I understand it the current cert. DST Root CA X3 Expiration (September 2021) is causing rasplex issues. Is there a way to update just the cert or does the whole project have to be updated? I'm assuming this is the issue since that's the error i get.

**Comment by dalehamel** [comment]
@Foresaken1 see https://github.com/RasPlex/RasPlex/issues/645#issuecomment-932594283 there are several workarounds.

TL;DR - the DST Root CA X3 cert expired, which is the root CA for LetsEncrypt which is what Plex uses. The new root CA that LetsEncrypt actually was already present in the certificate authority within Rasplex, but there is a bug in the older version of openssl (1.0.2k) that the image uses, which more recent openssl (1.1.0 and later, i believe) doesn't have any issue with. The problem is that the new cert was signed with this revoked cert, and so SSL validation fails, because it finds the root cert, and sees it is expired, and doesn't try the new cert, even though it has it.

The problem has been fixed by releasing new images which remove the cert, and there are workarounds posted for how to update existing installations, or do a server-side fix (which is a big security downgrade, but does fix the problem in many cases)
